Entity/User.php

```diff
@@ -69,7 +76,12 @@ public function getRoles()
      * This should be the encoded password. On authentication, a plain-text
      * password will be salted, encoded, and then compared to this value.
      *
-     * @return string The password
+     * The password will not be returned by this class. Symfony doesn't need
+     * the password of a phpBB user, as the User entity is read only.
+     *
+     * Any changes to the user should be made within phpBB.
+     *
+     * @return null The password
```

Comment:
`null The password`? I think just `null` would do here as null is not the password.